ftw/simplelayout/browser/client/dist/simplelayout.js

```diff
@@ -635,6 +635,13 @@ define('app/simplelayout/Block',[
       return this;
     };
 
+    this.markLinks = function() {
```

Comment:
As @shylux sait the `main.js` ist just for integration and the code in `Block.js` is testable as well.
The block knows how to mark its content through the `this.element` reference.
So I would leave the `markLinks` function here.